c2rust-analyze/src/rewrite/expr/distribute.rs

```diff
@@ -65,7 +65,7 @@ pub fn distribute(
             let origin = match unlower_map.get(&key) {
                 Some(x) => x,
                 None => {
-                    error!("unlower_map has no origin for {:?}", key);
+                    error!("unlower_map has no origin for {:?} {:?}", key, unlower_map);
```

Comment:
```suggestion
                    error!("unlower_map has no origin for {:?} {:?}", key);
```
Should this be reverted?  It looks like a debug print that would be incredibly verbose in normal usage.